src/EditorFeatures/Test2/Rename/RenameTagProducerTests.vb

```diff
@@ -565,8 +566,9 @@ class C
                 Dim textBuffer = workspace.Documents.Single().TextBuffer
                 Dim renameService = workspace.GetService(Of InlineRenameService)()
                 Dim editorOperations = workspace.GetService(Of IEditorOperationsFactoryService).GetEditorOperations(view)
-                Dim commandHandler As New RenameCommandHandler(workspace.GetService(Of InlineRenameService),
-                                                               workspace.GetService(Of IEditorOperationsFactoryService))
+                Dim commandHandler As New RenameCommandHandler(
+                    workspace.ExportProvider.GetExportedValue(Of IThreadingContext)(),
```

Comment:
Why mixing ExportProvider and GetService?

Reply:
`IThreadingContext` is exported directly (not a workspace service)